source/module_hamilt_lcao/module_gint/CMakeLists.txt

```diff
@@ -21,13 +24,14 @@ list(APPEND objects
     gint_force_cpu_interface.cpp
     gint_rho_cpu_interface.cpp
     gint_vl_cpu_interface.cpp
-    cal_psir_ylm.cpp
-    cal_dpsir_ylm.cpp
-    cal_ddpsir_ylm.cpp
-    mult_psi_dm.cpp
-    mult_psi_dm_new.cpp
-    mult_psi_dmr.cpp
-    init_orb.cpp
+    # cal_psir_ylm.cpp
```

Comment:
please just remove the deleted files from the list

Reply:
The issue has been fixed.

Reply:
The latest test network has issues. Do I need to push again?